CONTRIBUTING

```diff
@@ -0,0 +1,35 @@
+# Contributing
+
+Hey, thanks for your interest in contributing to the NYU CS Wiki! Please review this document **before making a pull request**.
```

Comment:
I think it would be good to make a note of a communication channel to ask questions/get some interactions.